services/core/MasterDriverAgent/tests/unit/test_driver.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from datetime import datetime, date, time
+
+import pytest
+
+from services.core.MasterDriverAgent.master_driver import agent
+from services.core.MasterDriverAgent.master_driver.agent import DriverAgent
```

Comment:
services.core should not be a module.  This should not work

Reply:
I initially imported this module because I was getting import error messages when Codacy ran Pylint. Are you implying that I should be importing from the MasterDriverAgent folder instead?